test/e2e/metrics/signature-approved.spec.js

```diff
@@ -0,0 +1,348 @@
+const { strict: assert } = require('assert');
+const {
+  convertToHexValue,
+  withFixtures,
+  regularDelayMs,
+  openDapp,
+  unlockWallet,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+/**
+ * Shared Ganache options for every test in this suite.
+ */
+const ganacheOptions = {
+  accounts: [
+    {
+      secretKey:
+        '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+      balance: convertToHexValue(25000000000000000000),
+    },
+  ],
+};
```

Comment:
You can use `defaultGanacheOptions` from `helpers` which has the account with 25eth in it :)

Reply:
Nice! Good call!